docs/faq.md

```diff
@@ -92,6 +92,13 @@ the python requirements do not have precompiled packages for ARM /
 ARM64. Installation of these will require additional development
 libraries and compilation will take a long time.
 
+!!! note
+
+    For ARMv7 systems, paperless may still function, but it could require
```

Comment:
Is it worth clarifying what this is e.g. 32bit ARM?

Reply:
Tweaked to add that